netsim/modules/lag.py

```diff
@@ -26,21 +26,27 @@ def populate_lag_id_set(topology: Box) -> None:
 """
 def create_lag_member_links(l: Box, topology: Box) -> None:
   lag_members = l.lag.members
-  l.lag.pop("members",None)                      # Remove explicit list of members
-  l2_ifdata = { 'type': "p2p", 'prefix': False } # Construct an L2 member link
+  l.lag.pop("members",None)                                    # Remove explicit list of members
+  l2_ifdata = data.get_box({ 'type': "p2p", 'prefix': False }) # Construct an L2 member link
```

Comment:
I can't figure out why you need this change.

Reply:
I removed ```lag.index``` from ```lag_l2_ifattr``` because it wasn't behaving the way I expected it to - it copied all the ```lag``` attributes. After removing that, the ```member.lag._parentindex = l.linkindex``` statement throws an error because it's trying to use '.lag' as an attribute